plugins/outputs/mongodb/mongodb.go

```diff
@@ -0,0 +1,189 @@
+package mongodb
+
+import (
+	"context"
+	"time"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/outputs"
+	"github.com/influxdata/telegraf/plugins/serializers"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+	"go.mongodb.org/mongo-driver/mongo/readpref"
+)
+
+type MongoDB struct {
+	serializer serializers.Serializer
+
+	URI             string   `toml:"uri"`
+	DB              string   `toml:"db"`
+	Collection      string   `toml:"collection"`
+	Timeout         uint64   `toml:"timeout"`
+	MaxPoolSize     uint64   `toml:"max_pool_size"`
+	MinPoolSize     uint64   `toml:"min_pool_size"`
+	MaxConnIdleTime uint64   `toml:"max_conn_idle_time"`
```

Comment:
You might want to use
```suggestion
	MaxConnIdleTime internal.Duration   `toml:"max_conn_idle_time"`
```
here to allow the specification of `max_conn_idle_time = "30s"` in the config.